content/en/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
 {{% /capture %}}
 
 {{% capture body %}}
-Kubernetes is a portable, extensible open-source platform for managing
-containerized workloads and services, that facilitates both
-declarative configuration and automation. It has a large, rapidly
-growing ecosystem. Kubernetes services, support, and tools are widely available.
+Kubernetes is a cloud infrastructure that provides for the deployment and
+orchestration of containerized applications.    
+It is a portable and extensible open-source platform that is capable of 
+managing both containerized workloads and services. Kubernetes
+provides key features such as a declarative configuration and deployment model,
+built in support for application replicas and autoscaling of applications, and
+also provides support for deploying multiple versions of an application.  
+In addition, Kubernetes has built in health-checking support for its applications,
+and networking and volume support as well. 
 
 Google open-sourced the Kubernetes project in 2014. Kubernetes builds upon
 a [decade and a half of experience that Google has with running
 production workloads at
 scale](https://research.google.com/pubs/pub43438.html), combined with
-best-of-breed ideas and practices from the community.
+best-of-breed ideas and practices from the community.  Kubernetes has a large 
+and rapidly growing ecosystem. Kubernetes is now available as a hosted managed 
+service from all major public cloud providers, and there are several distribution 
+options available for running Kubernetes as an on-premises private cloud as well.
+Furthermore, Kubernetes services, support, and tools are widely available.
```

Comment:
nit: does the clause `and there are several distribution options ...` help others understand what Kubernetes is?
nit: possibly reword or drop `Furthermore`
nit: does widely available mean: on any number of cloud platforms, OSes, computer languages?

Reply:
Will fix as requested by a previous reviewer